stdpopsim/catalog/homo_sapiens.py

```diff
@@ -149,6 +169,42 @@
     doi="https://doi.org/10.1126/science.1219240")
 
 
+def _constant():
```

Comment:
This is unnecessary I think --- we'll get this just by running ``stdpopsim homsap`` without specifying a model.

Reply:
Ah, you're right! This model is unnecessary